script-library/python-debian.sh

```diff
@@ -275,9 +275,40 @@ install_using_oryx() {
     ln -s ${PYTHON_INSTALL_PATH}/bin/python3-config ${PYTHON_INSTALL_PATH}/bin/python-config
 }
 
+link_existing_python() {
```

Comment:
Not sure if I even need to link this.

I'm only symlinking the binary here- I was going to do the whole python folder (with `python -c "import sys; print(sys.exec_prefix)`, but in some cases the system version is _just_ a binary in `/usr`.